pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -734,7 +734,7 @@ func (m *mockStore) Delete(k string) error {
 	return m.delete(k)
 }
 
-func (m *mockStore) Query(query string) (storage.StoreIterator, error) {
+func (m *mockStore) Query(indexKey string, indexValue string) (storage.StoreIterator, error) {
 	return m.query(query)
```

Comment:
`query` argument is not refactored following the signature change above

Reply:
Fixed